.gitignore

```diff
@@ -1,4 +1,5 @@
 articles/
+builds/*.phar
```

Comment:
The goal was to push dev versions to users who don't want to run the build script as suggested in the [README.md](https://github.com/designsecurity/progpilot/blob/master/README.md).

Two possibilities:
- remove `builds/*.phar` from `.gitignore` as before
- change the [README.md](https://github.com/designsecurity/progpilot/blob/master/README.md) to remove the link to [builds](./builds/) folder and suggest instead the execution of ./build.sh, something like that:
> Download the latest phar archive in [releases](https://github.com/designsecurity/progpilot/releases) folder (or [build](./build.sh) your own phar archive).